multivac/gather_job_data.py

```diff
@@ -175,15 +185,24 @@ def put_to_db(self):
         for job in list(self.gathered_data.keys()):
             curr_job = self.gathered_data[job]
 
-            # Convert string to a datetime object, then convert it to unix
-            # time (seconds), then to timestamp in nanoseconds.
-            time_to_datetime = datetime.fromisoformat(
-                f"{curr_job['queued_at'].rstrip('Z')}+00:00")
-            time_to_unix = datetime.timestamp(time_to_datetime)
-            unix_ns_time = int(time_to_unix * 1000000000)
+            if 'failure_type' in curr_job.keys():
```

Comment:
Looks good! Just an idea: what if we change `failure_type` to something like `status_details` and put in one of:

* `success`
* detected failure type
* `unknown failure`?

Reply:
UPD: decided that we don't need to do it, just change value `'unknown'` to `'unknown_failure'`

Reply:
```suggestion
measurement = curr_job['failure_type'] or curr_job['conclusion']
```

Reply:
Done